<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWebrtcClass</class>
 <widget class="QMainWindow" name="QtWebrtcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LicodeClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QWidget" name="controlWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>url</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEIP">
           <property name="text">
            <string>https://192.168.1.107:3004</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogin">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogout">
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>peer id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEPeerID">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConnectPeer">
           <property name="text">
            <string>connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnPreviewStart">
           <property name="text">
            <string>preview start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPreviewStop">
           <property name="text">
            <string>preview stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>是否发送音视频数据给对方，如果不发送，则对方没有声音和图像</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="btnSendEnable">
            <property name="text">
             <string>enable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSendDisable">
            <property name="text">
             <string>disable</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>摄像头</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QComboBox" name="cbCamera">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnCamreaRefresh">
                <property name="text">
                 <string>refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnCamreaOpen">
                <property name="text">
                 <string>open</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string>截屏，整个屏幕、特定区域</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QComboBox" name="cbScreenTitle">
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbScreenId">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnScreenRefresh">
                    <property name="text">
                     <string>refresh</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>left</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lEScreenLeft">
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>top</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLineEdit" name="lEScreenTop">
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>right</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="lEScreenRight">
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>bottom</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLineEdit" name="lEScreenBottom">
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0" colspan="2">
                     <widget class="QPushButton" name="btnScreenRect">
                      <property name="text">
                       <string>getRect</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rBRectScreen">
                    <property name="text">
                     <string>rect</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="rBFullScreen">
                    <property name="text">
                     <string>full</string>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btnScreenOpen">
                <property name="text">
                 <string>open</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>窗口</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QComboBox" name="cbWindowTitle">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbWindowId">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnWindowRefresh">
                <property name="text">
                 <string>refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnWindowOpen">
                <property name="text">
                 <string>open</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="text">
            <string>close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="renders" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QVideoRender" name="localRender" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QVideoRender" name="remoteRender_1" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QVideoRender" name="remoteRender_2" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QVideoRender" name="remoteRender_3" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoRender</class>
   <extends>QWidget</extends>
   <header>QVideoRender.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtWebrtc.qrc"/>
 </resources>
 <connections/>
</ui>
